src/DataProtection/Cryptography.KeyDerivation/src/Microsoft.AspNetCore.Cryptography.KeyDerivation.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core utilities for key derivation.</Description>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
```suggestion
    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Reply:
On second thought, this should probably stay the same as it was in 2.x and target `netstandard2.0;netcoreapp2.0`. By changing the TFMs to `netstandard2.0;netcoreapp3.0`, anyone who uses the 3.0 package on netcoreapp2.x will end up with different behavior because they will get the netstandard DLL, not the netcore DLL. This will have a negative perf impact on Linux/macOS.

cref https://github.com/aspnet/AspNetCore/commit/e552b5861a9f35a3daa3079d0e56b011f907daed